### ISSUE TITLE
Neural Translation Model example is not really learning French

### ISSUE THREAD

**Issue by eli7** [tensorflow/skflow#150]
I trained it for several hours with a big GPU (12GB) and all it could do on the test set was get at most the first word sort of right and then repeat some seemingly random word until the end of the sentence for instance (made up examples, but hopefully you get the idea):

Mr President advised the council to (...)

Translates to:
Monsieur rement rement rement rement rement rement rement

and 

He thought it was time for a change (...)

Translates to:
Il rement rement rement rement rement rement

Example code URL: https://github.com/tensorflow/skflow/blob/master/examples/neural_translation_word.py

**Comment by kaneplusplus** [comment]
+1

**Comment by Arthiv23** [comment]
Facing same issue. In my case it ran for several hours  ( 15 +  hours) but even first fit cycle did not complete.  Has anyone got this code working for translation?  If yes what is the translation accuracy.

**Comment by ilblackdragon** [comment]
@Arthiv23  We are revamping examples so will try to fix if there are any issues. Sorry that it took so long - it took awhile to merge all the pieces into main TensorFlow and align with other libraries.

Let me know if you are debugging though and see some clear issues.
For example, did the loss went down while training or stayed at the same value? How many steps per hour did it make?

If nothing else I'll ping with an update in a few days.

**Comment by ilblackdragon** [comment]
Inactive, if still issues - please re-file bug at tensorflow repository - this one is inactive. Thanks!
